.github/workflows/check-performance-regression.yml

```diff
@@ -47,4 +49,4 @@ jobs:
 
       - name: Run performance test
         run: |
-          ./.ci/check-performance-regression.sh
+          bash ./.ci/check-performance-regression.sh
```

Comment:
Why you need to run nested bash ?
It already in bash.

Reply:
@romani I was testing locally, and it turns out to be we need to use `bash` to have access to the `time` command we use. But I guess we don't need this anymore here in the real environment. 

I will remove it.

Reply:
I am waiting to see whether https://github.com/checkstyle/checkstyle/actions/runs/8945608285/job/24574850218?pr=14855 runs good.

Reply:
Removed